### PR TITLE
Properly set location on default rules

### PR DESCRIPTION
**Pull request by mmussomele**
Also removes a redundant location set on the body of default rules.

### CODE REVIEW

File: ast/rego.peg [hunk 1]
```diff
@@ -156,9 +156,9 @@ DefaultRules <- "default" ws name:Var _ "=" _ value:Term {
 
     loc := currentLocation(c)
     body := NewBody(NewExpr(BooleanTerm(true).SetLocation(loc)))
-    body[0].Location =loc
```
Comment (torin-styra) [inline ast/rego.peg]: The expression should also have a location set. I'd leave this line and just add the one below.

Comment (mmussomele) [inline ast/rego.peg]: It's set a few lines down again

Comment (mmussomele) [inline ast/rego.peg]: https://github.com/open-policy-agent/opa/pull/357/commits/3e55fde879a559e2502f23c051ceb8710d0db6c3#diff-3a3409dd6ff56d5dffc7ef0adfba9b33R170

Comment (tsandall) [inline ast/rego.peg]: Thanks!

**Pull request merged**